test/javascript/components/mentoring/DiscussionPostList.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { render, waitFor } from '@testing-library/react'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+import '@testing-library/jest-dom/extend-expect'
+import { DiscussionPostList } from '../../../../app/javascript/components/mentoring/DiscussionPostList'
+
+test('displays all posts', async () => {
```

Comment:
The test suggests that it checks if all posts are displayed, but only one post is returned. Would it make sense to add another post to the returned JSON and check if that is also rendered?

Reply:
Good idea. I've done this.